specfile/changelog.py

```diff
@@ -0,0 +1,127 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import collections
+import datetime
+import re
+from typing import List
+
+from specfile.sections import Section
+
+DAYS = [
+    "Mon",
+    "Tue",
+    "Wed",
+    "Thu",
+    "Fri",
+    "Sat",
+    "Sun",
+]
+
+MONTHS = [
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "Jul",
+    "Aug",
+    "Sep",
+    "Oct",
+    "Nov",
+    "Dec",
+]
+
+
+class ChangelogEntry:
+    """
+     Class that represents a changelog entry.
+
+    Attributes:
+        date: Date of the entry.
+        author: Author of the entry.
+        evr: EVR (epoch, version, release) of the entry.
+        content: List of lines forming the content of the entry.
+    """
+
+    def __init__(
+        self, date: datetime.date, author: str, evr: str, content: List[str]
+    ) -> None:
+        self.date = date
+        self.author = author
+        self.evr = evr
+        self.content = content.copy()
+
+    def __str__(self) -> str:
+        return f"{self.header}\n" + "\n".join(self.content) + "\n"
```

Comment:
Couldn't this type of joining cause some problems with extra blanks lines? I am not completely sure whether they are allowed in the changelog or not. (same for `Changelog.reassemble()`)

Reply:
What extra blank lines do you mean? Could you give an example?

Reply:
What I thought about here was something like:
```
* Mon Jun 22 2022 Author theauthor@mail.com
<blank line here>
- entry
- entry
```

Reply:
Yes, this is actually valid, RPM doesn't have any issues with it. However, this would happen only if user wanted to have blank lines there, am I right? In that case, why not just let him?

Reply:
Oh, good point, for some reason I thought that the `__str__` method was being used for reassembly similar to sections. Looking at it again, the case mentioned above should hopefully work just fine as far as I can see (there will be a single changelog entry with `content` equal to `["", "- entry", "-entry"]`)